Octokit/Models/Request/EditRepositoryHook.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Octokit.Internal;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class EditRepositoryHook
+    {
+        [Parameter(Key = "config")]
+        public dynamic Config { get; set; }
```

Comment:
Or could we make it a collection type that helps the user understand what sort of payload they're sending?